deployment/enterprise/sso/boxy.sql

```diff
@@ -0,0 +1,2 @@
+-- Bootstrap script for the database when first initialized
+CREATE DATABASE jackson;
```

Comment:
Can we extend this or mention in docs it is advised to also create dedicated pg user for this database?